blocks/init/src/Blocks/custom/column/manifest.json

```diff
@@ -167,5 +180,106 @@
 				"value": "bottom"
 			}
 		]
+	},
+	"variables": {
+		"columnWidth": [
+			{
+				"inverse": true,
+				"variable": {
+					"column-width": "%value%"
+				}
+			}
+		],
+		"columnOffset": [
+			{
+				"inverse": true,
+				"variable": {
+					"column-offset": "%value%"
+				}
+			}
+		],
+		"columnHide": {
+			"false": [
+				{
+					"inverse": true,
+					"variable": {
+						"column-display": "flex"
+					}
+				}
+			],
+			"true": [
+				{
+					"inverse": true,
+					"variable": {
+						"column-display": "none"
+					}
+				}
+			]
+		},
+		"columnOrder": [
+			{
+				"inverse": true,
+				"variable": {
+					"column-order": "%value%"
+				}
+			}
+		],
+		"columnVerticalAlign": {
```

Comment:
I've added horizontal align in Infinum web project. maybe it would be good to do the same here.